fastestimator/estimator.py

```diff
@@ -82,11 +83,14 @@ def __init__(self,
                  epochs: int,
                  train_steps_per_epoch: Optional[int] = None,
                  eval_steps_per_epoch: Optional[int] = None,
-                 traces: Union[None, Trace, Scheduler[Trace], Iterable[Union[Trace, Scheduler[Trace]]]] = None,
+                 traces: Union[None, Trace, Scheduler[Trace],
+                               Iterable[Union[Trace, Scheduler[Trace]]]] = None,
                  log_steps: Optional[int] = 100,
+                 eval_log_steps: List = [],
```

Comment:
default args in a function call shouldn't be mutable. This would need to be an empty tuple instead. Also, typing should indicate what kind of values are intended here (List[int] for example.).